src/app/geo-header/geo-header.component.ts

```diff
@@ -1,24 +1,24 @@
 import { Component, Output } from '@angular/core';
 import { Store } from '@ngrx/store';
-
 import { warnSearchResult } from '../state-management/actions/search-result-action';
 import { SearchingStates } from '../state-management/states/search-result-state';
 import { changeSearchFromAddress } from '../state-management/actions/current-search-action';
+import { GeolocationService, GeocodeService } from '../geo/geo.module';
 
 @Component({
   selector: 'app-geo-header',
   templateUrl: './geo-header.component.html',
-  styleUrls: ['./geo-header.component.scss']
+  styleUrls: [ './geo-header.component.scss' ]
 })
 export class GeoHeaderComponent {
+  @Output() geoAddress = '';
```

Comment:
I'm not familiar with this use of _@Output_ decorator. Shouldn't it be used to emit events to the parent components?